src/components/main-page/About/About.module.scss

```
@@ -0,0 +1,48 @@
+.cardContainer {
+  display: grid;
+  grid-template-columns: repeat(1, var(--card-width));
+  align-items: center;
+  justify-content: center;
+  justify-items: center;
+  row-gap: var(--space-l);
+}
+
+.textContainer {
+  margin-top: var(--space-l);
+  margin-bottom: var(--space-xl);
+}
+
+.textField {
+  text-align: center;
+  margin-bottom: var(--space-m);
+}
+
+.textField:last-of-type {
+  margin-bottom: 0;
+}
+
+// tablet
+@media (min-width: 480px) {
+  .cardContainer {
+    grid-template-columns: repeat(2, var(--card-width));
+    gap: var(--space-2xl) var(--space-2xl);
```

if the value is the same, can't we just
```suggestion
    gap: var(--space-2xl);
```
?